src/FSharp.AWS.DynamoDB/TableContext.fs

```diff
@@ -76,16 +76,18 @@ type TableContext<'TRecord> internal
 
         let! ct = Async.CancellationToken
         let! response = client.GetItemAsync(request, ct) |> Async.AwaitTaskCorrect
+        match maybeReport with None -> () | Some r -> r GetItem [ response.ConsumedCapacity ] (if response.IsItemSet then 1 else 0)
```

Comment:
@samritchie I feel the predecessor PR should move all the reporting steps to right after the core call - the main objective should be to trap every cost, not get correct reports as to successful outcomes